README.md

```diff
@@ -552,6 +554,50 @@ block-beta
 protoflow execute WriteStdout < input.txt > output.txt
 ```
 
+#### [`ConcatStrings`]
+
+A block for concatenating all string messages it receives
+
+```mermaid
+block-beta
+    columns 7
+    Source space:2 ConcatStrings space:2 Sink
+    Source-- "input" -->ConcatStrings
+    ConcatStrings-- "output" -->Sink
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class ConcatStrings block
+    class Source hidden
+    class Sink hidden
+```
+
+```bash
+protoflow execute ConcatStrings joiner=","
+```
+
+#### [`SplitString`]
+
+A block that splits string
```

Comment:
This description should be improved, and should generally be the same or better as given in the table above.